Sources/SpeziOpenAI/MessageView.swift

```diff
@@ -14,6 +14,7 @@ import SwiftUI
 public struct MessageView: View {
     let chat: Chat
     let hideSystemMessages: Bool
+    let hideFunctionMessages: Bool
```

Comment:
I think it would rather make sense to make the API extensible for the future and even beyond different messages. E.g. using a dynamic set of roles could be a good way to enable the functionality:
```suggestion
    let hideMessagesWithRoles: Set<Chat.Role>
```